src/cmakeProject.ts

```diff
@@ -545,7 +579,11 @@ export class CMakeProject {
         for (const sub of [this.generatorSub, this.preferredGeneratorsSub, this.communicationModeSub, this.sourceDirSub]) {
             sub.dispose();
         }
+        this.kitsController.dispose();
```

Comment:
if the kitscontroller is shared by multiple projects in a workspaceFolder, then this would not be the correct place to dispose of it.